.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,12 @@
-Fixes #
+Fixes #<!-- Insert issue number that this PR fixes just after #-->  
```

Comment:
Put on a separate line? I don't think that many people know that this is a comment and can either be removed or is ignored.